src/pages/About/About.js

```diff
@@ -7,37 +8,33 @@ import styled from 'styled-components';
 const AboutPageLayout = styled.div`
   display: grid;
   grid-template-rows: auto auto;
-  margin-top: 5vh;
-`
+  justify-content: center;
+  margin: 5vh 0;
+` 
 
-const TeamCardGrid =  styled.div`
+const TeamCardGrid =  styled.div` 
   display: grid;
-  grid-template-columns: auto auto auto auto;
+  grid-template-columns: auto auto auto;
   grid-column-gap: 20px;
-  margin-left: auto;
-  margin-right: auto;
+  grid-row-gap: 20px;
 `
 
-const WhoWeAre = styled.div`
-  display: block;
-  font-size: 27px;
-  text-align: center;
-  width: 80vw;
-  height: 10rem;
-  padding: 20px;
-  margin-left: auto;
-  margin-right: auto;
-  margin-bottom: 50px;
-`
+// const WhoWeAre = styled.div`
```

Comment:
Why did you comment out the code instead of deleting? Are you expecting for it to be used somewhere else in the project?

Reply:
True, i thought I'd be adding that block later. I'll delete it